docs/overview/Lifecycle.md

```diff
@@ -0,0 +1,383 @@
+<!--ONE-DOCS-HIDE start-->
+[👈 Return to Overview](./README.md)
+<!--ONE-DOCS-HIDE end-->
+
+[one-app-dev-cdn]: https://github.com/americanexpress/one-app-dev-cdn
+[one-app-dev-proxy]: https://github.com/americanexpress/one-app-dev-proxy
+[one-app-runner]: https://github.com/americanexpress/one-app-cli/tree/main/packages/one-app-runner
+[one-app-router]: https://github.com/americanexpress/one-app-router
+[one-app-router-route]: https://github.com/americanexpress/one-app-router/blob/main/docs/API.md#route
+[one-app-router-router]: https://github.com/americanexpress/one-app-router/blob/main/docs/API.md#router
+[one-app-ducks]: https://github.com/americanexpress/one-app-ducks
+[render-partial-only]: https://github.com/americanexpress/one-app-ducks#setrenderpartialonly
+[render-text-only]: https://github.com/americanexpress/one-app-ducks#setrendertextonly
+[holocron-module-route]: https://github.com/americanexpress/holocron/tree/main/packages/holocron-module-route#moduleroute
+[create-holocron-store]: https://github.com/americanexpress/holocron/tree/main/packages/holocron/docs/api#createholocronstore
+[render-module]: https://github.com/americanexpress/holocron/tree/main/packages/holocron/docs/api#rendermodule
+[react-helmet]: https://github.com/nfl/react-helmet
+[helmet]: https://github.com/helmetjs/helmet
+
+# One App Life Cycles
+
+In this guide, we'll break down what happens when One App starts up,
+renders a Holocron module and how we can configure the behavior of
+One App (and it's life cycles) via configuration.
+
+## 📖 Table of Contents
+* [One App Server](#one-app-server)
+  * [Server Settings](#server-settings)
+  * [Holocron Runtime](#holocron-runtime)
+* [Runtime App Configuration](#runtime-app-configuration)
+* [Server Side Rendering](#server-side-rendering)
+  * [Rendering Life Cycle](#rendering-life-cycle)
+  * [Partials Rendering](#partials-rendering)
+  * [Text Rendering](#text-rendering)
+
+## One App Server
+
+When the One App server first boots up, there are a series of steps
+undertaken before the server becomes operational - configuring One App
+based on environment settings is the first step, followed by pre-loading
+all the Holocron modules onto the server.
+
+### Server Settings
```

Comment:
This very useful but is more to do with configuring the one app server than understanding the lifecycle. This could probably be moved elsewhere and referenced here

Reply:
Agree, will do.

Reply:
This was a good call IMO, I split the guide into lifecycle and configuration (7e618c0)